lib/container_orchestrator.rb

```diff
@@ -8,22 +8,41 @@ class ContainerOrchestrator
   TOKEN_FILE   = "/run/secrets/kubernetes.io/serviceaccount/token".freeze
   CA_CERT_FILE = "/run/secrets/kubernetes.io/serviceaccount/ca.crt".freeze
 
+  class_attribute :cached_deployment, :default => Hash.new({})
```

Comment:
Where does the other cache live?  I expected them to be side-by-side (or even the same Hash)

Reply:
I can move it near the pod monitor code in kubernetes.rb above.

I would not recommend mixing them in the same hash due this comment [below](https://github.com/ManageIQ/manageiq/pull/20865/files#r536144688).

Reply:
I have moved the caching to the pod monitor code